### DISCUSSION TITLE
matplotlib click event

### DISCUSSION THREAD

**Discussion by MPenaR** [jupyterlite/jupyterlite#435] [Q&A]
Does jupyterlite support matplotlib click events? 

And it that case, how can I use it? 

Thank you very much

**Comment by bollwyvl** [comment]
Dunno! Try hacking the examples below and see if they do what you want!
 
https://jupyterlite.readthedocs.io/en/latest/_static/lab?path=pyolite%2Fmatplotlib.ipynb&path=pyolite%2Fipympl.ipynb

If it works and it's neat, we're always looking for suggestions to improve the examples.

**Reply by MPenaR** [reply]
If I try to do `%matplotlib notebook` I get a:
```
FileNotFoundError: [Errno 44] No such file or directory: '/lib/python3.9/site-packages/matplotlib/backends/web_backend/js/mpl.js'
```
I guess that means it doesn't work

**Reply by bollwyvl** [reply]
Do you have an example from the upstream matplotlib/pyodide docs of what you are actually trying to accomplish? Matplotlib is rather a complex beast, and part of the reason for pyodide in the first place, but not every feature is fully supported, especially one that relies on specific renderers.

Further, because of the constraints of serving potentially multiple (kinds of) kernels in JupyterLite, we use pyodide inside a webworker instead of the main UI thread, as blocking everything felt... bad. So if there are are specific features _only_ supported by magic pyodide stuff, they might not work either.
